src/Sulu/Bundle/ContentBundle/Resources/public/js/components/content-datasource/main.js

```diff
@@ -69,7 +70,10 @@ define([], function() {
                 {
                     el: this.$columnNavigationElement,
                     instanceName: 'smart-content-' + this.options.instanceName,
-                    url: this.prepareUrl(this.options.url),
+                    url: this.getUrl(),
+                    linkedName: '_linked',
+                    typeName: '_type',
```

Comment:
For me it doesn't feel like this... It's specific to this resource, and doesn't have a big impact on the entire system. /cc @chirimoya